app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+  validates :name, :surname, presence: true,
+                             length: { in: 3..30 },
+                             format: { with: /\A[a-zA-Z]+\z/,
+                                       message: 'only allows letters' }
+
+  validates :phone_number, presence: true,
+                           numericality: true,
+                           length: { is: 13 }
```

Comment:
Here use two spaces too

Reply:
I don't have to use here 2 spaces per indentation level too :)

Reply:
https://github.com/bbatsov/ruby-style-guide#no-double-indent
Both are considered as good style, in this case, please decide with your team and pick a style that everyone will follow